<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDlg</class>
 <widget class="QDialog" name="SettingDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/new/prefix1/images/setting.png</normaloff>:/new/prefix1/images/setting.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Grid</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Measurement's unit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbCm">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Cm</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbInch">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Inch</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Grid's step</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="dsp1">
          <property name="minimum">
           <double>0.500000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="chShowGrid">
        <property name="text">
         <string>Show grid</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Internationalization</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="lblLanguage">
          <property name="text">
           <string>Language:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbLanguage">
          <item>
           <property name="text">
            <string>System Default</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Arabic عربي</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>American English</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Chinese</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Dutch</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>French</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Georgian ქართული</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>German</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Italian</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Portuguese</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Russian Русский</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Serbian</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Serbian Latin</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Spanish</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Tamil தமிழ்</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ukraine Український</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkUpdates">
     <property name="text">
      <string>Check updates during start application</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>SettingDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSave</sender>
   <signal>clicked()</signal>
   <receiver>SettingDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
